account_move_line_stock_info/tests/test_account_move_line_stock_move.py

```diff
@@ -0,0 +1,186 @@
+# © 2015 Eficent Business and IT Consulting Services S.L. (www.eficent.com)
```

Comment:
Many thanks for migrating this. We have a new copyright now. Could you change it please?

`Copyright 2019 ForgeFlow S.L. (https://www.forgeflow.com)`

Thanks in advance

Reply:
Changed.